compose/foundation/foundation/src/desktopTest/kotlin/androidx/compose/foundation/gestures/DesktopScrollableTest.kt

```diff
@@ -232,6 +233,47 @@ class DesktopScrollableTest {
         assertThat(column.offset).isEqualTo(0f)
     }
 
+    @Test
+    fun smoothScrollingDisabled() = runSkikoComposeUiTest(
+        size = size,
+        density = density
+    ) {
+        with(LinuxGnomeConfig) {
+            // Enabled by default
+            assertTrue(isSmoothScrollingEnabled)
+            isSmoothScrollingEnabled = false
+        }
+
+        val context = TestColumn()
+        setContent {
+            CompositionLocalProvider(
+                LocalScrollConfig provides LinuxGnomeConfig
+            ) {
+                Box(
+                    Modifier
+                        .scrollable(
+                            orientation = Orientation.Vertical,
+                            state = context.controller()
+                        )
+                        .size(10.dp, 20.dp)
+                )
+            }
+        }
+
+        scene.sendPointerEvent(
+            eventType = PointerEventType.Scroll,
+            position = Offset.Zero,
+            scrollDelta = Offset(0f, -5f),
+            nativeEvent = awtWheelEvent(),
+        )
+
+        // Check without waitForIdle
+        assertThat(context.offset).isWithin(0.1f).of(5f * scrollLineLinux(20.dp))
+
+        // Restore state
+        LinuxGnomeConfig.isSmoothScrollingEnabled = true
```

Comment:
We should wrap it into try-finally. Otherwise if this test fails others will fail as well.

Reply:
> others will fail as well.

Actually no, but let's do it to avoid extra questions/doubts